src/psyclone/tests/psyir/nodes/structure_reference_test.py

```diff
@@ -118,8 +118,8 @@ def test_struc_ref_create_errors():
     ''' Tests for the validation checks in the create method. '''
     with pytest.raises(TypeError) as err:
         _ = nodes.StructureReference.create(None, [])
-    assert ("'symbol' argument to StructureReference.create() should be a "
-            "DataSymbol but found 'NoneType'" in str(err.value))
+    assert ("A StructureReference must refer to a symbol that is (or could be)"
+            " a structure, has been given a 'None' with name: 'unknown'")
```

Comment:
"...but has been given a ..."

Reply:
Ups, it was missing the "in" operator, and didn't spot that the original already had the "but"

Reply:
Oh yes, almost as good as Joerg's test :-)